lib/shared/routers/public.js

```diff
@@ -3,22 +3,46 @@ import Public from 'screens/public';
 import React from 'react';
 import SchemaEntry from 'screens/public/screens/schema-entry';
 import {Route, IndexRoute} from 'react-router';
+import request from 'helpers/request';
+
+function initialized (nextState, replaceState, callback) {
```

Comment:
This logic should be in the server side route https://github.com/relax/relax/blob/master/lib/server/routers/auth.js#L79 this way this route is never accessible.